start project 3

mandunow opened the pull request.

Commit:
start project 3

Commit:
🏂🎩 Checkpoint
./index.html:22212454/1141

Commit:
📍👽 Checkpoint
./index.html:22212454/1234

Commit:
💸🕰 Checkpoint
./index.html:22212454/456

Commit:
🍢🦀 Checkpoint
./index.html:22212454/12938

angeliquejw commented on index.html:
```diff
@@ -0,0 +1,66 @@
+      </ul>
+    </nav>
+    <main>
+      <p id="About">
```
Instead of using `p` elements here, consider putting your `id`s on `section` elements and then having one or more `p` elements inside of that?

angeliquejw commented on index.html:
```diff
@@ -0,0 +1,66 @@
+        10:30 A.M.: Hand Lettering workshop 1:00 P.M.: Typeface as a design tool
+        2:30 P.M.: Visual Typography
+      </p>
+      <p id="Speakers">
```
This is a good example of the note I made above -- you probably want a separate `p` for each of the speakers. And possibly also some additional styling/emphasis on their names (which could be handled via a `strong` element or a `span`).

angeliquejw commented on index.html:
```diff
@@ -0,0 +1,66 @@
+        <li><a href="#Register">Register</a></li>
+      </ul>
+    </nav>
+    <main>
```
✅ This is a required element, good job!

angeliquejw commented on index.html:
```diff
@@ -0,0 +1,66 @@
+      
+    </main>
+  </body>
+</html>
```
I know your HTML is still something a work in progress, so I'm just listing the requirements here to help you along:

The design must include:
-    Event name ✅ I expect this will end up in your `h1` on line 12
-    In-page navigation ✅ 
-    Event schedule ✅ 
-    Details for at least three talks (including talk name and description) ◻️
-    Three speaker bios (including a photo) ✅ Photo still to be added
-    Call to action and link to buy tickets ◻️

Your code must include the following HTML elements:
- header ✅ 
- main ✅ 
- footer
- h1 ✅ 
- img
- a ✅

angeliquejw reviewed (commented):
Hi, Arielle -- Thanks for sharing your markup with me! I've left a few notes and outlined where you stand re: the project 3 requirements. Let me know if you have any questions or if I can help out in any way!

Commit:
🌄📇 Checkpoint
./index.html:22212454/98

Commit:
🍪🏟 Checkpoint
./index.html:22212454/157

Commit:
💪👣 Checkpoint
./index.html:22212454/6704

Commit:
🎯👯 Checkpoint
./index.html:22212454/385

Commit:
🚣👟 Checkpoint
./index.html:22212454/416

Commit:
🙈🐩 Checkpoint
./styles.css:22212454/33
./index.html:22212454/75

Commit:
💐👼 Checkpoint
./index.html:22212454/219

Commit:
🌔📒 Checkpoint
./index.html:22212454/411

Commit:
🚃🐊 Checkpoint
./index.html:22212454/9414

Commit:
🌷🏀 Checkpoint
./index.html:22212454/10

Commit:
🚙🤘 Checkpoint
./index.html:22212454/534
./styles.css:22212454/937

Commit:
👭🎥 Checkpoint
./index.html:22212454/203
./styles.css:22212454/50

Commit:
👂🏦 Checkpoint
./styles.css:22212454/72

Commit:
📮⚡️ Checkpoint
./styles.css:22212454/35

Commit:
💅🍀 Checkpoint
./styles.css:22212454/80

Commit:
👭😽 Checkpoint
./styles.css:22212454/113
./index.html:22212454/301

Commit:
🍿🚩 Checkpoint
./styles.css:22212454/97

Commit:
🎪🙇 Checkpoint
./index.html:22212454/697

Commit:
👽🏈 Checkpoint
./.glitch-assets:22212454/367
./index.html:22212454/140
./styles.css:22212454/1583

Commit:
👠🍠 Checkpoint
./styles.css:22212454/424
./index.html:22212454/263

Commit:
🚦✨ Checkpoint
./index.html:22212454/163

Commit:
✋💓 Checkpoint
./index.html:22212454/22463

Commit:
🚕🚓 Checkpoint
./index.html:22212454/115

Commit:
📮🏁 Checkpoint
./index.html:22212454/29
./styles.css:22212454/436
./menu.js:22212454/447

Commit:
🗾💃 Checkpoint
./styles.css:22212454/254
./.glitch-assets:22212454/1221
./index.html:22212454/11504

Commit:
🎯⛪️ Checkpoint
./styles.css:22212454/361

Commit:
🚧👠 Checkpoint
./styles.css:22212454/2816

Commit:
🐦🚘 Checkpoint
./styles.css:22212454/302
./index.html:22212454/23849

Commit:
🍫👮 Checkpoint
./styles.css:22212454/2729
./index.html:22212454/30

Commit:
🎺🚕 Checkpoint
./index.html:22212454/1054
./styles.css:22212454/802

Commit:
🐿🔮 Checkpoint
./styles.css:22212454/805

Commit:
🧀👚 Checkpoint
./styles.css:22212454/406
./index.html:22212454/6

Commit:
⛄️🗽 Checkpoint
./styles.css:22212454/6629
./index.html:22212454/14

Commit:
🌴😎 Checkpoint
./styles.css:22212454/181

Commit:
👽👡 Checkpoint
./styles.css:22212454/508
./index.html:22212454/15

Commit:
🏢⛳️ Checkpoint
./index.html:22212454/12865
./styles.css:22212454/4298

Commit:
🚲🏝 Checkpoint
./styles.css:22212454/472

Commit:
📷🍓 Checkpoint
./styles.css:22212454/735
./index.html:22212454/74

Commit:
💨⛄️ Checkpoint
./styles.css:22212454/681
./index.html:22212454/19

Commit:
👶🚓 Checkpoint
./styles.css:22212454/1555

Commit:
🛎🍧 Checkpoint
./index.html:22212454/10
./styles.css:22212454/6814

Commit:
🛬🚌 Checkpoint
./styles.css:22212454/292
./index.html:22212454/38

Commit:
🐀🎥 Checkpoint
./styles.css:22212454/374
./index.html:22212454/231

Commit:
👩🌒 Checkpoint
./styles.css:22212454/856

Commit:
🚧⛺️ Checkpoint
./styles.css:22212454/6524

Commit:
🌕🌖 Checkpoint
./styles.css:22212454/6715

Commit:
🐡👢 Checkpoint
./styles.css:22212454/411
./index.html:22212454/188

Commit:
🐂🎣 Checkpoint
./styles.css:22212454/472

Commit:
🌟🍈 Checkpoint
./styles.css:22212454/408
./index.html:22212454/188

Commit:
done with proj 3

Commit:
📠🐬 Checkpoint
./index.html:22212454/197
./styles.css:22212454/196

Commit:
🏭🌧 Checkpoint
./styles.css:22212454/8564
